include/mbedtls/platform_util.h

```diff
@@ -275,8 +275,8 @@ uint32_t mbedtls_platform_random_in_range( uint32_t num );
  *              Duration of the delay is random as number of variable increments
  *              is randomized.
  *
- * \note        Currently the function is dependent of hardware providing an
- *              rng with MBEDTLS_ENTROPY_HARDWARE_ALT.
+ * \note        This function depends on whether the MBEDTLS_FI_COUNTERMEASURES
```

Comment:
Please expand on how it depends on this flag. Will it stop working without it? Does it alter its behaviour?

Reply:
It works only I the MBEDTLS_FI_COUNTERMEASURES is defined. I added an explanation to the header file.